APIs/common/paths.py

```diff
@@ -0,0 +1,2 @@
+LOGGING_PATH = 'RUN THE SETUP SCRIPT AND THESE PATHS WILL BE OVERWRITTEN'
+DB_PATH = 'RUN THE SETUP SCRIPT AND THESE PATHS WILL BE OVERWRITTEN'
```

Comment:
Does this refer to the setup script for the pi?

Reply:
It refers to setup.sh